src/components/CertificateTemplates/tlds/sg/edu/ntu/common/certificate.js

```diff
@@ -0,0 +1,314 @@
+import { get } from "lodash";
+import { IMG_LOGO_NTU, IMG_CERTIFICATE_SEAL } from "./images";
+
+import { formatDate, formatCertID } from "./functions";
+
+export const fullWidthStyle = {
+  width: "100%",
+  height: "auto"
+};
+
+export const fullWidthStyleNTUText = {
+  width: "70%",
+  height: "auto"
+};
+
+export const fullWidthStyleSignature = {
+  width: "80%",
+  height: "auto"
+};
+
+export const certificateDimension = {
+  width: "100%",
+  height: "auto"
+};
+
+export const hrLineStyle = {
+  backgroundColor: "black"
+};
+
+export const printTextStyle = {
+  fontFamily: "Avant Garde,Avantgarde,Century Gothic,CenturyGothic,AppleGothic",
```

Comment:
i'm not sure these are web-safe fonts, i suspect they might not look as you intend on computers without these fonts

![image](https://user-images.githubusercontent.com/27843274/63733595-83efbb80-c8ab-11e9-824b-a99d6667c7ee.png)

Is this correct?

Reply:
I have verified with business users and they have no issues with this.